src/launch/lib/service/template/functions.py

```diff
@@ -110,12 +113,11 @@ def copy_template_files(src_dir: Path, target_dir: Path, dry_run: bool = True) -
         target_item = os.path.join(target_dir, item)
         if os.path.isdir(src_item):
             if (
-                item != PLATFORM_SRC_DIR_PATH
-                and item not in DISCOVERY_FORBIDDEN_DIRECTORIES
-            ):
+                item != PLATFORM_SRC_DIR_PATH or not_platform
+            ) and item not in DISCOVERY_FORBIDDEN_DIRECTORIES:
                 shutil.copytree(src_item, target_item, dirs_exist_ok=True)
         else:
-            shutil.copy2(src_item, target_item)
+            shutil.copy2(src_item, target_item, follow_symlinks=False)
```

Comment:
Is the idea here that if a user has a symlink to a source template file, we want to copy the symlink rather than the contents?